src/pybind/mgr/volumes/fs/operations/clone_index.py

```diff
@@ -47,6 +47,42 @@ def untrack(self, tracking_id):
         except cephfs.Error as e:
             raise IndexException(-e.args[0], e.args[1])
 
+    def get_all_entries(self):
+        dir_entries = get_all_dir_entries(self.fs, self.path)
+        # TODO: what if all entries were created in same second?
+        # stat command gets milli-seconds as well, that will be useful -
+        # Access: 2024-02-12 14:31:56.678240755 +0530
```

Comment:
That's fine, you'd need not be super accurate when picking up entries in ctime order. If 4 clones indexes have the same ctime (seconds granularity), let those get picked up.